src/treebuilders/project.cpp

```diff
@@ -55,14 +54,11 @@ namespace mrcpp {
  * A negative maxIter means no bound.
  *
  */
+
 template <int D>
-void project(double prec,
-             FunctionTree<D> &out,
-             std::function<double(const Coord<D> &r)> func,
-             int maxIter,
-             bool absPrec) {
+void project(double prec, FunctionTree<D> &out, std::function<double(const Coord<D> &r)> func, int maxIter) {
```

Comment:
What happened to `absPrec`? Are you sure your branch is up to date with latest `master`?

Reply:
I did a `git rebase` and it gave me no conflicts :/